.packit.yaml

```diff
@@ -0,0 +1,24 @@
+files_to_sync:
+  - dnf5.spec
+
+jobs:
+  - job: propose_downstream
+    trigger: release
+    # Use the stage instance once it works in downstream.
```

Comment:
Sorry, but I don't understand the comment. Could you please clarify?

Reply:
/me copypasting with no coffee today. sorry, I it's fixed now